src/app/child-dev-project/children/dashboard-widgets/birthday-dashboard/birthday-dashboard.component.html

```diff
@@ -0,0 +1,62 @@
+<!--
+  TODO
+  - Look for a better icon here https://fontawesome.com/v5/search
+  - Set right title, subtitle, explanation
+-->
+<app-dashboard-widget
+  icon="birthday-cake"
+  theme="child"
+  title="Birthday"
+  subtitle="Upcoming Birthdays"
+  explanation="Temporary explanation"
+>
+  <app-widget-content>
+    
+    <div *ngIf="childrendataSource.data.length > 0 && !isLoading" class="table-wrapper">
+      <table mat-table [dataSource]="childrendataSource" matSort>
+        <tr hidden="true">
+          <th scope="col" mat-sort-header>Child </th>
+          <th scope="col" mat-sort-header>DateOfBirth</th>
+          <th scope="col" mat-sort-header>AfterBirthdayAge</th>
+        </tr>
+        <ng-container matColumnDef="entityId">
+          <td mat-cell *matCellDef="let child">{{ child.entityId}} 
+          </td>
+        </ng-container>
+
+        <ng-container matColumnDef="dateOfBirth">
+          <td mat-cell *matCellDef="let child">{{child.dateOfBirth}} </td>
+        </ng-container>
+
+        <ng-container matColumnDef="age">
+          <td mat-cell *matCellDef="let child">{{child.age}}</td>
+        </ng-container>
+
+    
+<!--
+  TODO
+  - replace this by a table similar to `AttendanceWeekDashboard` (for more info see https://v11.material.angular.io/components/table/overview)
+  - display the child using the `ChildBlockComponent`
+  - display the birthday
+  - display the age the child will have after its birthday
+   
+-->
+           <app-child-block
+             *ngFor="let child of children"
+             [entity]="child"
+           ></app-child-block>
+           <tr mat-header-row *matHeaderRowDef="displayedColumns"></tr>
```

Comment:
```suggestion
```
This is only necessary if you have a `*matHeaderCellDef` in your template. Otherwise it will look for this and throw an error because it cannot find it.